src/app/login/login.page.html

```diff
@@ -0,0 +1,61 @@
+<ion-content>
+  <div class="background"><div class="background-inner"></div></div>
+  <div class="header">
+    <h1 class="max-wrap ion-padding">
+      Volunteer Login
+    </h1>
+  </div>
+
+  <div class="max-wrap">
+    <ion-card>
+      <ion-card-content>
+        <form #form="ngForm" (ngSubmit)="signin(form)">
+          <ion-list>
+            <ion-item>
+              <ion-label position="stacked"
+                >Email<span class="required">*</span></ion-label
+              >
+              <ion-input
+                [(ngModel)]="form.email"
+                name="email"
+                required="true"
+                placeholder="Your email address"
+              ></ion-input>
+              <!--  <ion-text
+                [hidden]="ngForm.controls.email && (ngForm.controls.email.valid || (ngForm.controls.email.pristine))"
+                color="danger"
+                >Email is required</ion-text
+              > -->
+            </ion-item>
+            <ion-item>
+              <ion-label position="stacked"
+                >Password<span class="required">*</span></ion-label
+              >
+              <ion-input
+                [(ngModel)]="form.password"
+                name="password"
+                required="true"
+                placeholder="Your password"
+              ></ion-input>
+              <!-- <ion-text
+                [hidden]="ngForm.controls.password && (ngForm.controls.password.valid || (ngForm.controls.password.pristine))"
+                color="danger"
+                >Password is required</ion-text
+              > -->
+            </ion-item>
+          </ion-list>
+        </form>
+        <ion-button [routerLink]="">Login</ion-button>
```

Comment:
The button should be inside the form. Then the routerLink property should be removed. The button will automatically submit the form (and trigger the onSubmit action in the form tag).